Focusgroup/explainer.md

```diff
@@ -140,12 +159,63 @@ Example 1:
 </p>
 ```
 
-For the `<p>` element which defines the above focusgroup, the elements "one", "two", and "three"
-can be focused using the arrow keys (up/right moves focus forward, down/left moves focus backwards).
-Note that only elements "one" and "three" can be focused using the Tab key (because element "two" has
-`tabindex=-1` set, which takes it out of the tabindex sequential navigation ordering).
+Using CSS, a focusgroup definition can be applied with selectors, and must include a name. `auto` is 
+a reserved name that corresponds to the same focusgroup implied by the HTML `focusgroup` attribute.
+
+```css
+#parent { 
+   focus-group: auto;
+}
+```
+
+For the `parent` element which includes the focusgroup definition, the elements `one`, `two`, and `three`
+(and any other children of `parent` that may be added or removed) are focusgroup candidates and because
+each are focusable, they also become focusgroup items. When one of the focusgroup items is focused, then
+the user can move focus sequentially according to DOM order among all the focusgroup items using the arrow
+keys (up/right moves focus forward, down/left moves focus backwards). Note that only elements `one` and
+`three` can be focused using the Tab key (because element `two` has `tabindex=-1` set, which takes it out
+of the tabindex sequential navigation ordering).
+
+Focusgroup definitions may include the following (in addition to a name):
+
+* extend -- a mechanism to join this focus group to an ancestor focusgroup. Note: linear focusgroups cannot
```

Comment:
Might want to specify that extend should only be used on linear focusgroups, not grid ones. Just like you did with the direction value a couple of lines below.